src/System.Xml.XPath.XmlDocument/src/System.Xml.XPath.XmlDocument.csproj

```
@@ -9,9 +9,4 @@
     <VersionPrefix Condition="'$(IsPackable)' == 'true'">4.4.0</VersionPrefix>
     <PackageValidationBaselineVersion>4.3.0</PackageValidationBaselineVersion>
   </PropertyGroup>
-
-  <ItemGroup>
-    <Compile Include="$(CommonPath)src\System\NotImplemented.cs" />
-  </ItemGroup>
-
 </Project>
```

nit: please keep the empty line before the project closing tag for readability

```suggestion

</Project>
```